src/ops/fuelup_toolchain/uninstall.rs

```diff
@@ -20,13 +20,17 @@ pub fn uninstall(command: UninstallCommand) -> Result<()> {
     };
 
     if !toolchain.exists() {
-        info!("toolchain '{}' does not exist", &toolchain.name);
+        println_warn(format!("Toolchain '{}' does not exist", &toolchain.name));
         return Ok(());
     }
 
+    if config.list_toolchains()?.len() == 1 {
```

Comment:
From this comment: https://github.com/FuelLabs/fuelup/pull/574/files#r1495080029

This should be something like:

```
    if toolchain.name == settings.default_toolchain {
```